source/fundamentals/authentication.txt

```diff
@@ -1 +1,208 @@
-.. _csharp-authentication-mechanisms:
\ No newline at end of file
+.. _csharp-authentication-mechanisms:
+
+=========================
+Authentication Mechanisms
+=========================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you can learn how to authenticate with MongoDB using each
+**authentication mechanism** available in the {+mongo-community+}.
+Authentication mechanisms are processes by which the driver and server confirm
+identity and establish trust to ensure security before connecting.
+
+You can use the following authentication mechanisms with the latest version
+of {+mongo-community+}:
+
+- :ref:`csharp-scram-sha-256`
+- :ref:`csharp-scram-sha-1`
+- :ref:`csharp-x509`
+
+To authenticate using ``Kerberos`` or ``LDAP``, see the
+:ref:`csharp-enterprise-authentication-mechanisms` guide. For more information
```

Comment:
this ref is showing up empty in the build. If the page isnt populated yet, maybe add a title to the shell page? or leave this out for now.

Reply:
Added a title to the shell page for now